ITEM.tsv

```diff
@@ -398,206 +398,206 @@ ITEM_20150317_000397	$Viper
-ITEM_20150317_000582	Some Vubbes do carry useful weapons. 
-ITEM_20150317_000583	Deathweaver Club 
-ITEM_20150317_000584	A moment of silence for the previous owner who became the sacrifice required to make this Deathweaver. 
-ITEM_20150317_000585	Chapparition Mace 
-ITEM_20150317_000586	No matter what monster you face, this weapon will leave a dent. 
-ITEM_20150317_000587	Stunner 
-ITEM_20150317_000588	Some clerics prefer to disorient their target through means other than preaching. 
-ITEM_20150317_000589	Valia 
-ITEM_20150317_000590	It is consecrated with the power of the Goddess during the crafting process. Elder Demons can't stand the sight of this weapon. 
-ITEM_20150317_000591	Spearfish Rod 
-ITEM_20150317_000592	May it produce barbed dicipline. 
-ITEM_20150317_000593	Wooden Shield 
-ITEM_20150317_000594	Though made of wood, this has been crafted to meet the minimum requirements as a shield. 
-ITEM_20150317_000595	Wooden Buckler 
-ITEM_20150317_000596	Surprisingly, the act of reinforcing the buckler with a metal rim is only a recent development. Historians presume this trend first started around the founding period of the Kingdom. 
-ITEM_20150317_000597	Wooden Kite Shield 
-ITEM_20150317_000598	After examining a shield he improvised during an emergency, the Peltasta Master refined it into a new, multipurpose shield. 
-ITEM_20150317_000599	Kite Shield 
-ITEM_20150317_000600	The metal kite shield used by the Cavalry has been further improved and enhanced by the warriors who preferred sturdiness over solid weight.  
+ITEM_20150317_000401	$Chapparition Bow 
+ITEM_20150317_000402	$The engravings say: 'Straighten your back and broaden your shoulders; pull the bowstring and push the bow as you shoot.' 
+ITEM_20150317_000403	$Blood Gain 
+ITEM_20150317_000404	$This bow was created by Lydia Schaffen. She frequently used this bow throughout her lifetime but intentionally witheld the crafting method from 'The Tower of the Stars' and 'Schaffenstar', two entities founded by Lydia herself. 
+ITEM_20150317_000405	$Tempest Shooter 
+ITEM_20150317_000406	$A new bow designed and developed through a joint effor of master bowcrafters. It was conceived with the hope of overcoming the Great Plant Cataclysm. 
+ITEM_20150317_000407	$Empty_weapon_Bow 
+ITEM_20150317_000408	$Low Light Crossbow 
+ITEM_20150317_000409	$Crossbow [Stab] 
+ITEM_20150317_000410	$Useful when having a free hand is more important than having more attack power. 
+ITEM_20150317_000411	$Low Crossbow 
+ITEM_20150317_000412	$Modern crossbows use many different materials, mostly monster parts. 
+ITEM_20150317_000413	$Dokyu 
+ITEM_20150317_000414	$A weapon that is easy to reload. 
+ITEM_20150317_000415	$Quarrel Bow 
+ITEM_20150317_000416	$The Quarrel Shooter Master uses this to teach beginners. 
+ITEM_20150317_000417	$Oak Crossbow 
+ITEM_20150317_000418	$Although craftable wood is hard to find since the Great Plant Cataclysm, this weapon is not hard to make. 
```

Comment:
Suggest a change of tense:
"Although craftable wood has been hard to find since the Great Plant Cataclysm, this weapon is not hard to make."
or a change of timing:
"Although craftable wood is hard to find because of the Great Plant Cataclysm, this weapon is not hard to make."